translation/japanese/patterns/transparent-cross-team-decision-making-using-rfcs.md

```diff
@@ -0,0 +1,127 @@
+## Title
+
+RFCを用いたチーム横断的な意思決定の透明化
+
+## Patlet
+
+高い参加率を達成し、関係者全員にとって最良の意思決定を行いたいインナーソースプロジェクトは、ソフトウェアのライフサイクル全体を通して参加型のシステムを構築する方法を見つける必要があります。内部のRFC(Requests for Comments)ドキュメントを公開することで、設計プロセスの早い段階から議論を行うことができ、関係者全員が高いコミットメントを持ってソリューションを構築できる可能性が高まります。
+
+## 問題
+
+インナーソースプロジェクトが健全であるためには、相当な数のコントリビューターが必要です。しかし、コントリビュータがー増えると、例えば互いに互換性のない機能をプロジェクトに追加したり、アーキテクチャを不健全に肥大化させたりする可能性があります。
```

Comment:
```suggestion
インナーソースプロジェクトが健全であるためには、相当な数のコントリビューターが必要です。しかし、コントリビューターが増えると、例えば互いに互換性のない機能をプロジェクトに追加したり、アーキテクチャを不健全に肥大化させたりする可能性があります。
```

Reply:
Thank you for the correction, I think it is good to unify the katakana notation